src/broker/rusage.c

```diff
@@ -31,10 +31,7 @@
 #include "rusage.h"
 
 struct rusage_context {
-    flux_msg_handler_t *w;
-};
-
-static void rusage_request_cb (flux_t *h, flux_msg_handler_t *w,
+    flux_msg_handler_t *mh; }; static void rusage_request_cb (flux_t *h, flux_msg_handler_t *mh,
```

Comment:
something got messed up here, accidental backspace put these on the same line?

Reply:
Oops!  Sheesh.  How'd I miss that.

Just force pushed a fix to that one.